doc/release-notes/release-7.sgml.in

```diff
@@ -103,6 +104,36 @@ The Squid-@SQUID_RELEASE@ change history can be <url url="https://github.com/squ
 <p>See <url url="https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes"> for
    the full ISO-639 list. HTTP uses the 2-letter (set 1) codes.
 
+<sect1>Forwarded HTTP Extension support
+<p>Details in <url url="https://tools.ietf.org/html/rfc7239" name="RFC 7239: Forwarded HTTP Extension">
+
+<p>The <em>Forwarded</em> extension has been standardized to replace
+   <em>X-Forwarded-For</em> and similar custom headers. This release contains
+   initial support for managing this HTTP extension in Squid with the
+   <em>http_header_forwarded</em> directive.
+
+<p>The default behaviour is <em>transparent</em> - to relay any <em>Forwarded</em>
+   or <em>X-Forwarded-For</em> headers from the client unchanged to the server.
+   Unlike <em>X-Forwarded-For</em> the default is not to expose client IP in the
+   <em>Forwarded</em>.
```

Comment:
Headers themselves do not have defaults. We need to explicitly say (somewhere in these release notes) that we are changing default Squid behavior (from adding an XFF header field with the client IP address to not adding an XFF or Forwarded header field with the client IP address, even though the received XFF/Forwarded fields are still forwarded by default).